.buildkite/nightly-benchmarks/tests/raw-json-header.md

```diff
@@ -0,0 +1,6 @@
+
+## Raw json data
+
+This section contains raw experimental results in json format, and the data of markdown tables above in json format.
+
+You can use this json data to conviniently load the benchmarking results into a program. This data will also be used to generate performance visualizations in the upcoming vllm dashboard.
```

Comment:
why do we need so many markdown files instread of just one?

Reply:
The current benchmarking will interleave these markdown files with the benchmarking results, like:
[latency-test-header.md]
[latency benchmarking tables]
[throughput-test-header.md]
[throughput benchmarking tables]
It is easier to do so with all the markdown files separated.